components/profiler/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "profiler"
+version = "0.0.1"
+edition = "2018"
+publish = false
+
+[features]
+profiling = ["lazy_static", "cpuprofiler", "callgrind", "valgrind_request"]
```

Comment:
Can it compile on Windows without the feature enabled?

Reply:
It should work though I don't have a Windows machine. I changed it to `[target.'cfg(linux)'.dependencies]` and it can compile on my MacOS.